metricbeat/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '2.1'
```

Comment:
This change shouldn't go here

Reply:
This is needed to define `healthchecks` in docker compose files.

Reply:
What I mean is that if you want to change version of this, it should be done in a different PR with some information about what could be affected with this version change. This is the main compose file for metricbeat, not directly docker related.

Reply:
Using 2.1 is perfectly fine, we already use different versions along beats, I don't think that opening a PR just changing this line contribute much.

Reply:
All beats uses `2.3` and the only Beat using `2` is Metricbeat. According to this: https://docs.docker.com/compose/compose-file/compose-versioning/#version-23 version `2.3` uses Docker Engine `v17.06.0+` and it's only supported by Compose `v1.16.0+`.

So it seems that the reasonble step would be to upgrade to `2.3` to match the rest of the beats but ensuring (explicitly) that versions of Docker Engine and Compose are covered by the CI and the build system. We can _guess_ that it's going to work because all Beats are using this version. What's the Compose and Docker Engine version that we are using right now?

Reply:
PR created for upgrade to 2.3 https://github.com/elastic/beats/pull/10868

We use docker-compose 1.23.2 and in CI we have docker engine 17.09.0-ce in travis and 18.06-ce in jenkins.